packages/perseus/src/util.ts

```diff
@@ -146,9 +146,15 @@ function firstNumericalParse(text: string): ParsedValue | null | undefined {
 }
 
 function stringArrayOfSize(size: number): ReadonlyArray<string> {
-    return _(size).times(function () {
-        return "";
-    });
+    return Array(size).fill("");
+}
+
+function stringArrayOfSize2D(
+    rows: number,
+    columns: number,
+): ReadonlyArray<ReadonlyArray<string>> {
+    const rowArr = stringArrayOfSize(rows);
```

Comment:
You can do
```suggestion
    const rowArr = Array(rows).fill();
```

which might be slightly clearer (since you don't need the strings)

Reply:
`fill` requires an argument. I could do `.fill(null)` (which I'm not sure is clearer than `.fill("")`) but I like leveraging the existing helper.